src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -0,0 +1,97 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    background-color: var(--color-black);
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  background-color: var(--color-black);
+  color: var(--color-white);
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: repeat(2, auto);
+  padding-bottom: 150px;
+  padding-left: 75px;
+  padding-top: 100px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 78px;
+    padding-left: 0;
+    padding-top: 50px;
+  }
+}
+
+.allTextWrap {
+  grid-column-end: 11;
+  grid-column-start: 2;
+}
+
+.gridAreaBottom {
+  display: flex;
+  flex-direction: row;
+  grid-row: 2;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaBottom {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.title {
+  font: var(--font-title);
+  margin: 0;
+  padding-bottom: 54px;
```

Comment:
Please use margin to create spacing between elements, not padding. Padding is "part of" the element; margin is "outside of" the element.

To demonstrate, you can temporarily set a border on this element and see where it gets drawn, where the border represents the boundary of the element. With padding-bottom, the border gets drawn around the padding, which doesn't look right because it doesn't match one thinks of being "part of the element" when looking at the design.

Reply:
Also note that this spacing is not correct for mobile, so you will need to override it in a media query.

Reply:
👍 Changed to margin. Corrected spacing on mobile view.